src/lib/y2partitioner/widgets/lvm_vg_devices_selector.rb

```diff
@@ -129,12 +132,23 @@ def check_for_committed_devices(sids)
         return if committed_devices.empty?
 
         error_message = if committed_devices.size > 1
-          _("Physical volumes %s cannot be removed because they already exist on disk")
+          # TRANSLATORS: Error message when several physical volumes cannot be removed, where
+          # %{pvs} is replaced by device names (e.g., /dev/sda1, /dev/sda2) and %{vg} is
+          # replaced by the volume group name (e.g., system)
+          _("Removing physical volumes %{pvs} from the volume group %{vg}\n" \
+            "is not supported because the physical volume is already in use")
         else
-          _("Physical volume %s cannot be removed because it already exists on disk")
+          # TRANSLATORS: Error message when a physical volume cannot be removed, where
+          # %{pvs} is replaced by device name (e.g., /dev/sda1) and %{vg} is replaced
+          # by the volume group name (e.g., system)
+          _("Removing the physical volume %{pvs} from the volume group %{vg}\n" \
+            "is not supported because the physical volume is already in use")
```

Comment:
Also plural at the end of the sentence. On the other hand, for both messages I guess it's more precise to say "the physical volumes may be already in use".  Strictly speaking, we are doing it to stay safe, but maybe there is no information allocated.